.github/workflows/cd-on-push.yml

```
@@ -0,0 +1,97 @@
+name: 👉 Deploy on push
+
+on:
+  # !! Uncomment the following lines once you've set up the dev environment and ready to turn on continuous deployment
+  push:
+    branches: ["main", "PRP-166-redo"]
```

should remove this list? 
```suggestion
    branches: ["main"]
```